docs/zh/engines/database-engines/mysql.md

```diff
@@ -10,67 +15,86 @@ MySQL引擎用于将远程的MySQL服务器中的表映射到ClickHouse中，并
-| UNSIGNED INT, UNSIGNED MEDIUMINT | [UInt32](../../sql-reference/data-types/int-uint.md)        |
-| INT, MEDIUMINT                   | [Int32](../../sql-reference/data-types/int-uint.md)         |
-| UNSIGNED BIGINT                  | [UInt64](../../sql-reference/data-types/int-uint.md)        |
-| BIGINT                           | [Int64](../../sql-reference/data-types/int-uint.md)         |
-| FLOAT                            | [Float32](../../sql-reference/data-types/float.md)          |
-| DOUBLE                           | [Float64](../../sql-reference/data-types/float.md)          |
-| DATE                             | [Date](../../sql-reference/data-types/date.md)              |
-| DATETIME, TIMESTAMP              | [DateTime](../../sql-reference/data-types/datetime.md)      |
+**引擎参数**
+
+-   `host:port` — MySQL服务地址
+-   `database` — MySQL数据库名称
+-   `user` — MySQL用户名
+-   `password` — MySQL用户密码
+
+## 支持的数据类型 {#data_types-support}
+
+| MySQL                            | ClickHouse                                                   |
+|----------------------------------|--------------------------------------------------------------|
+| UNSIGNED TINYINT                 | [UInt8](../../sql-reference/data-types/int-uint.md)          |
+| TINYINT                          | [Int8](../../sql-reference/data-types/int-uint.md)           |
+| UNSIGNED SMALLINT                | [UInt16](../../sql-reference/data-types/int-uint.md)         |
+| SMALLINT                         | [Int16](../../sql-reference/data-types/int-uint.md)          |
+| UNSIGNED INT, UNSIGNED MEDIUMINT | [UInt32](../../sql-reference/data-types/int-uint.md)         |
+| INT, MEDIUMINT                   | [Int32](../../sql-reference/data-types/int-uint.md)          |
+| UNSIGNED BIGINT                  | [UInt64](../../sql-reference/data-types/int-uint.md)         |
+| BIGINT                           | [Int64](../../sql-reference/data-types/int-uint.md)          |
+| FLOAT                            | [Float32](../../sql-reference/data-types/float.md)           |
+| DOUBLE                           | [Float64](../../sql-reference/data-types/float.md)           |
+| DATE                             | [Date](../../sql-reference/data-types/date.md)               |
+| DATETIME, TIMESTAMP              | [DateTime](../../sql-reference/data-types/datetime.md)       |
 | BINARY                           | [FixedString](../../sql-reference/data-types/fixedstring.md) |
 
-其他的MySQL数据类型将全部都转换为[String](../../sql-reference/data-types/string.md)。
+其他的MySQL数据类型将全部都转换为[String](../../sql-reference/data-types/string.md).
+
+[Nullable](../../sql-reference/data-types/nullable.md)已经支持
+
+## 全局变量支持 {#global-variables-support}
+
+为了更好地兼容，您可以在SQL样式中设置全局变量，如`@@identifier`. 
 
-同时以上的所有类型都支持[Nullable](../../sql-reference/data-types/nullable.md)。
+支持这些变量：
+- `version`
+- `max_allowed_packet`
 
-## 使用示例 {#shi-yong-shi-li}
+!!! warning "Warning"
```

Comment:
Quoted words should be translated as the reader see them -> `!!! warning "(translation of warning)"`.